test/test_iperf.ml

```diff
@@ -194,42 +194,42 @@ let test_tcp_iperf_two_stacks_basic amt timeout () =
   let module Test = Test_iperf (Vnetif_backends.Basic) in
   Test.default_network () >>= fun { backend; Test.client; Test.server } ->
   Test.V.record_pcap backend
-    (Printf.sprintf "tests/pcap/tcp_iperf_two_stacks_basic_%d.pcap" amt)
+    (Printf.sprintf "tcp_iperf_two_stacks_basic_%d.pcap" amt)
     (Test.tcp_iperf ~server ~client amt timeout)
 
 let test_tcp_iperf_two_stacks_mtu amt timeout () =
   let module Test = Test_iperf (Vnetif_backends.Mtu_enforced) in
   Test.default_network () >>= fun { backend; Test.client; Test.server } ->
   Test.V.record_pcap backend
-    (Printf.sprintf "tests/pcap/tcp_iperf_two_stacks_mtu_%d.pcap" amt)
+    (Printf.sprintf "tcp_iperf_two_stacks_mtu_%d.pcap" amt)
```

Comment:
@yomimono it is not possible to add exta CLI arguments to alcotest yet, but it should be doable without much trouble. Would you mind opening an issue in the alcotest repo to keep track of that? I might add this shortly as I will need to make a new jbuilder release anyway.